src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Apprentice/Price.cshtml

```diff
@@ -0,0 +1,34 @@
+﻿@using SFA.DAS.CommitmentsV2.Shared.Extensions
+@using SFA.DAS.ProviderCommitments.Web.RouteValues
+@model SFA.DAS.ProviderCommitments.Web.Models.Apprentice.PriceViewModel
+
+@{
+    ViewBag.Title = "What's the new agreed apprenticeship price?";
+    ViewBag.GaData.Vpv = "/apprentices/change-employer/price";
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <partial name="_ValidationSummary" />
+        <h1 class="govuk-heading-xl">@ViewBag.Title</h1>
+        <p>This is the price agreed between you and the new employer to complete the remainder of the apprenticeship training and assessment.</p>
+        <form method="post">
+            @Html.HiddenFor(x => x.EmployerAccountLegalEntityPublicHashedId)
+            @Html.HiddenFor(x => x.StartDate)
+            <div class="govuk-form-group  @Html.AddClassIfPropertyInError(x => x.Price, "govuk-form-group--error ")">
+                @Html.ValidationMessageFor(m => m.Price, null, new { @class = "govuk-error-message", id = "error-message-" + Html.IdFor(m => m.Price) })
```

Comment:
If I enter an invalid value (for example, the letter "X"), the summary shows _two_ error messages - one for the invalid value (looks like an inbuilt MVC binding error) _and_ a custom "required field" message (because after binding, we haven't got a value on the vm so it looks like the user didn't enter one).

Reply:
Having said that, we need to look at the wider application and ensure we are consistent. I tried the Add Apprentice page and found that if you enter XXX in price it simply ignores it!